src/TLuaInterpreter.cpp

```diff
@@ -949,6 +949,21 @@ int TLuaInterpreter::getTextFormat(lua_State* L)
     return 1;
 }
 
+int TLuaInterpreter::getWindowsCodepage(lua_State* L)
+{
+#if defined (Q_OS_WIN32)
+    QSettings registry(QStringLiteral(R"(HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\Nls\CodePage)"),
+                       QSettings::NativeFormat);
+    auto value = registry.value(QStringLiteral("ACP"));
+    lua_pushstring(L, value.toString().toUtf8().constData());
+    return 1;
+#else
+    lua_pushnil(L);
+    lua_pushstring(L, "this function is only needed, and only works on Windows");
```

Comment:
"this function is only needed on Windows, and does not work here"

Reply:
This getWindowsCodepage() is available for all Lua users, right? Then we may need to translate its feedback.

Reply:
I'll adjust the text. It's a message returned vis script do it won't be translated